src/migration.ts

```diff
@@ -139,7 +139,11 @@ export class Migration implements RunMigration {
         ? resolve(process.cwd(), options.templateFileName)
         : resolve(
             __dirname,
-            `../templates/migration-template.${await resolveSuffix(directory, options)}`
+            join(
```

Comment:
Doesn't work with `esm` because `__dirname` changes from `/dist` to `/dist/esm`:

- **cjs**: `/node_modules/node-pg-migrate/templates` ✅
- **esm**: `/node_modules/node-pg-migrate/dist/templates` ❌

A possible solution is what I did in #1189:

```js
    const moduleDir = dirname(
      // @ts-expect-error: ignore until esm only
      createRequire(import.meta.url || __dirname).resolve(
        'node-pg-migrate-exp/package.json'
      )
    );
// ...
      : resolve(
            moduleDir,
            `templates/migration-template.${await resolveSuffix(directory, options)}`
          );

```

or

```js
      : resolve(
            'node_modules/node-pg-migrate/templates',
            `migration-template.${await resolveSuffix(directory, options)}`
          );

```

Reply:
Does this also work for pnpm? (I mean when there is a `node_modules/.pnpm` folder structure.)

Reply:
Yes, both suggestions work with pnpm links